docs/tutorial/events.js

```diff
@@ -143,20 +127,15 @@ Promise.all([dataLoaded, mdChartLoaded]).then((results) => {
           });
         },
         (chart) => {
-          chart.on("background-draw", backgroundImageHandler);
-          chart.render.updateFrame(true);
-          chart = chart.animate(
-            {
-              style: {
-                plot: {
-                  xAxis: { interlacing: { color: "#ffffff00" } },
-                  yAxis: { interlacing: { color: "#ffffff00" } },
-                },
-              },
-            },
-            { duration: 0 }
-          );
-          chart.then((chart) => chart.render.updateFrame(true));
+          if (!image.complete) {
```

Comment:
This would be more compact:
```
const registerHandler = () => {
  chart.on("background-draw", backgroundImageHandler);
  chart.render.updateFrame(true);
};
if (!image.complete) image.onload = registerHandler;
else registerHandler()
```